src/test/java/io/nats/client/impl/JetStreamPushQueueTests.java

```diff
@@ -156,33 +156,28 @@ public void testQueueSubErrors() throws Exception {
             js.subscribe(SUBJECT, pso1);
 
             IllegalArgumentException iae = assertThrows(IllegalArgumentException.class, () -> js.subscribe(SUBJECT, pso1));
-            String expected = String.format("Consumer [%s] is already bound to a subscription.", durable(1));
-            assertEquals(expected, iae.getMessage());
+            assertTrue(iae.getMessage().contains("[SUB-Q02]"));
```

Comment:
added "error codes" to the errors to make it possible to validate without checking the full text of the message